internal/uvm/create.go

```diff
@@ -247,3 +252,9 @@ func (uvm *UtilityVM) normalizeMemorySize(ctx context.Context, requested int32)
 	}
 	return actual
 }
+
+// FullyPhysicallyBacked returns if the UVM has been set to be entirely physically
+// backed including with additional devices
+func (uvm *UtilityVM) FullyPhysicallyBacked() bool {
```

Comment:
I think at the annotation level, we want to use fully physically backed, but on the UVM options and on the UVM itself, we should actually separately track the concepts of the UVM's memory backing, and the devices backing. So I would propose changing this to be `DevicesPhysicallyBacked` here.

Reply:
I like that idea. Really makes it clear that the annotation is just a meta annotation for other options. Will make that change :)

Reply:
Updated. I maintained the field for FullyPhysicallyBacked on the Options struct to ensure we have a set location for setting the default value.